src/style.css

```diff
@@ -0,0 +1,117 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Do you really need to reset these styles for all elements?

Reply:
fixed  and noted to future